### PR TITLE
feat(#major); Monitor; Create initial monitor script

### PR DESCRIPTION
**Pull request by MichaelC1999**
Note: rebased "Monitor" branch to dis-include unnecessary commit history. The original (closed, non-merged) PR with reviews is here https://github.com/messari/subgraphs/pull/826

### CODE REVIEW

File: monitor/lendingPoolLevel.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+        const instance = data[idx];
+        const deploymentName = Object.keys(deployments).find(
+            (depoName) => deployments[depoName].url === protocol.url
+        );
+        const deployment = { ...deployments[deploymentName] };
+
+        const issuesArrays = { ...deployment.poolErrors };
+
+        const buildIssue = (value) => protocol.deployment + "-" + instance.id + "-" + instance.name + "//" + value;
```
Comment (MichaelC1999) [inline monitor/lendingPoolLevel.js:81]: @jihok created function in for loop scope to build issue string

File: monitor/script.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+async function executionFlow() {
+  const { data } = await axios.get(
+    "https://subgraphs.messari.io/deployments.json"
+  );
+
+  // deployments holds the errors for every protocol
+  let deployments = {};
+
+  Object.entries(data).forEach(([protocolType, protocolsOnType]) => {
```
Comment (MichaelC1999) [inline monitor/script.js:20]: @jihok Changed Object.keys to Object.entries where appropriate

File: monitor/script.js [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+  await Promise.all(indexingStatusQueries)
+    .then(
+      (response) =>
+      (indexData = response.map(
+        (resultData) => (resultData.data.data)
+      ))
+    )
+    .catch((err) => console.log(err));
+
+  indexData = { ...indexData[0], ...indexData[1] };
```
Comment (MichaelC1999) [inline monitor/script.js:153]: @jihok 
https://github.com/messari/subgraphs/pull/826#discussion_r952162468
Before this logic, indexData is an array with two elements; an object of results from the current deployments query and an object of results from the pending deployments query. Before there was a loop here that would settle this into one object saved to indexData. This logic now condenses this into one line with a spread operator.

**Review by jihok — APPROVED** [review]
lgtm!

**Comment by this-username-is-taken** [comment]
Great work!

**Pull request merged**